auth-next/link-multiple-accounts.js

```diff
@@ -182,3 +182,61 @@ function unlink(providerId) {
   });
   // [END auth_unlink_provider]
 }
+
+function accountExistsPopup(auth, facebookProvider, goToApp, promptUserForPassword, promptUserForSignInMethod, getProviderForProviderId) {
+  // [START account_exists_popup]
+  const { signInWithPopup, signInWithEmailAndPassword, linkWithCredential } = require("firebase/auth");
+
+  // User tries to sign in with Facebook.
+  signInWithPopup(auth, facebookProvider).catch((error) => {
+  // User's email already exists.
+  if (error.code === 'auth/account-exists-with-different-credential') {
+    // The pending Facebook credential.
+    const pendingCred = error.credential;
+    // The provider account's email address.
+    const email = error.customData.email;
+    
+    // Present the user with a list of providers they might have
+    // used to create the original account.
+    // Then, ask the user to sign in with the existing provider.
+    const method = promptUserForSignInMethod();
+      
+    if (method === 'password') {
+      // TODO: Ask the user for their password.
```

Comment:
still needed?

Reply:
This was from the existing snippet. I think this is to tell developers to implement `promptUserForPassword` so let's keep it.